src/js/services/srv-asset.js

```diff
@@ -0,0 +1,12 @@
+const app = angular.module('materia')
+app.service('AssetSrv', function () {
+	const deleteAsset = (asset) =>
+		Materia.Coms.Json.del(`/api/asset/delete/${encodeURIComponent(asset.id)}`, asset)
```

Comment:
This will generate a `DELETE` request to `/api/asset/delete/#` where `#` is the asset's ID, but it will also (theoretically) send the entire asset object as the body of the request, which would make it unnecessary to include the asset's ID in the URL.

If all you need in the backend is the asset ID, don't bother passing the `asset` object along to `Materia.Coms.Json.del`. If you need more than the asset ID, don't bother making the asset's ID part of the request URL.